examples/bridge-app/linux/main.cpp

```diff
@@ -478,6 +549,44 @@ EmberAfStatus HandleReadSwitchAttribute(DeviceSwitch * dev, chip::AttributeId at
     return EMBER_ZCL_STATUS_SUCCESS;
 }
 
+EmberAfStatus HandleReadPowerSourceAttribute(DevicePowerSource * dev, chip::AttributeId attributeId, uint8_t * buffer,
+                                             uint16_t maxReadLength)
+{
+    if ((attributeId == ZCL_POWER_SOURCE_BAT_CHARGE_LEVEL_ATTRIBUTE_ID) && (maxReadLength == 1))
+    {
+        *buffer = dev->GetBatChargeLevel();
+    }
+    else if ((attributeId == ZCL_POWER_SOURCE_ORDER_ATTRIBUTE_ID) && (maxReadLength == 1))
+    {
+        *buffer = dev->GetOrder();
+    }
+    else if ((attributeId == ZCL_POWER_SOURCE_STATUS_ATTRIBUTE_ID) && (maxReadLength == 1))
+    {
+        *buffer = dev->GetStatus();
+    }
+    else if ((attributeId == ZCL_POWER_SOURCE_DESCRIPTION_ATTRIBUTE_ID) && (maxReadLength == kPowerSourceDescriptionSize))
+    {
+        uint8_t bufferMemory[kPowerSourceDescriptionSize];
+        MutableByteSpan zclString(bufferMemory);
+        MakeZclCharString(zclString, dev->GetDescription().c_str());
+        buffer = zclString.data();
+    }
+    else if ((attributeId == ZCL_CLUSTER_REVISION_SERVER_ATTRIBUTE_ID) && (maxReadLength == 2))
+    {
+        *buffer = (uint16_t) ZCL_POWER_SOURCE_CLUSTER_REVISION;
```

Comment:
This doesn't make sense, and I'm a little surprised it compiles at all.  Is this code being compiled without `-Wconversion -Werror` or something?

Reply:
I got mistake and fix it.
There are no changes in compiler options. This code was compiled with `-Wconversion  -Wall -Werror -Wextra`. Without any warnings or errors. So strange.